freeipa-client/src/main/java/com/sequenceiq/freeipa/client/RetryableFreeIpaClientException.java

```diff
@@ -35,4 +35,9 @@ public RetryableFreeIpaClientException(String message, RetryableFreeIpaClientExc
     public Exception getExceptionForRestApi() {
         return exceptionForRestApi;
     }
+
+    @Override
+    public String toString() {
+        return super.toString() + System.lineSeparator() + "ExceptionForRestApi: "  + exceptionForRestApi.toString();
```

Comment:
@jamisonbennett does this change is useful in your opinion? I just want to be sure we won't miss anything when logging this exception

Reply:
I think that should catch everything.